cogs/command_utils/games_util.py

```diff
@@ -0,0 +1,396 @@
+            """Calculates winner
+            If resulting winner is player_one then returns 0
+            If resulting winnner is player_two returns 1
+            if tied returns 2
+            """
+            if (second_player % 3) + 1 == first_player:
+                return 0
+            elif (first_player % 3) + 1 == second_player:
+                return 1
+            else:
+                return 2
+
+        responses = self.player_responses.values()
+        plys = [key for key in self.player_responses.keys()]
+        self.reset_responses()
+
+        result = calc_win(*responses)
+        player_one_words, player_two_words = await self.rps_to_words(*responses)
+        if result is 0:
+            return True, f"{plys[0]}'s {player_one_words} beats {plys[1]}'s {player_two_words}"
+        elif result is 1:
+            return False, f"{plys[0]}'s {player_one_words} is countered by {plys[1]}'s {player_two_words}"
+        else:
+            return None, False
+
+    async def embed_controller(self, create=False, send_or_await="", *args, sp=False):
+        if sp is False:
+            player = self.peek_player()
+        else:
+            player = self.players[sp]
+        await player.outfit_embed(self.format_string(send_or_await, *args))
+
+        if create is False:
+            try:
+                await player.message.edit(embed=player.emb)
+            except discord.Forbidden:
+                raise commands.BotMissingPermissions("Embed Links!")
+        else:
+            try:
+                player.message = await player.user.send(embed=player.emb)
+            except discord.Forbidden:
+                raise commands.BotMissingPermissions(f"DM {player.user.mention}")
+
+            for user_input_buttons in self.react_controls:
+                try:
+                    await player.message.add_reaction(user_input_buttons[0])
+                except (discord.NotFound, discord.Forbidden):
+                    pass
+
+    async def send_win_lose(self, win_lose):
+        if win_lose is True:
+            first_data, second_data = "win", "lose"
+
+        elif win_lose is False:
+            first_data, second_data = "lose", "win"
+        else:
+            first_data, second_data = "tie", "tie"
+
+        await self.players[0].update_footer(self.win_lose, self.current_round)
+        await self.players[1].update_footer(self.win_lose[::-1], self.current_round)
+        await self.embed_controller(False, first_data, self.players[0].user.mention, sp=0)
+        await self.embed_controller(False, second_data, self.players[1].user.mention, sp=1)
+
+    async def send_await(self):
+        wait = None  # Placeholder
+        for player, response in self.player_responses.items():
+            if response is None:
+                wait = player
+                break
+        for _ in self.players:
+            await self.embed_controller(False, "await", wait)
+
+    def invoker(self, reaction):
```

Comment:
C0116: Missing Function or method dosctring
R1710: Either all return statements in a function should return an expression, or none of them should